### ISSUE TITLE
Problem with stock_availible activation

### ISSUE THREAD

**Issue by ofrmax** [OCA/stock-logistics-warehouse#328]
Hi Guys,

I'm really new to odoo and I have a problem with the addon stock_availible.

I manage to install the addon on the server and also install it on odoo. But I'm facing an issue when i try to setup the addon ( in the setting menu of the inventory). There is 2 checkbox i can use. When I check both of them or only 1 odoo is loading and after bring me to app store page ( don't know why ). after going back to the setting page of the inventory, the checkbox is no longer activated. can you please help me identify why I have this behavior?

thanks in advance

**Comment by pedrobaeza** [comment]
Checkboxes? This module is not installed through checkboxes. Please install it as any other module in the app list.

**Comment by ofrmax** [comment]
Hi Pedro,

thanks for your quick answer. Unfortunately i dont think you understand my problem. The addons as been install in the app list but is not working properly. here is the checkboxes im talking about 

![image](https://user-images.githubusercontent.com/19998638/29326108-479696d6-81b8-11e7-9537-3cc6cefbb7d6.png)


When i check one or both of them and i click on apply. Odoo bring me to this page (really weird)

![image](https://user-images.githubusercontent.com/19998638/29326170-7cb08d54-81b8-11e7-98a8-329b3059bcab.png)


when I return to the setting the checkboxes has return uncheck


best regards,

**Comment by pedrobaeza** [comment]
OK, now you have explained it better. Another good thing to always say is your Odoo version.

**Comment by ofrmax** [comment]
here is my odoo version (10)

![image](https://user-images.githubusercontent.com/19998638/29326349-2fcc76f0-81b9-11e7-8b38-795538f6410e.png)

**Comment by ofrmax** [comment]
Can you please reopen the issue ?

**Comment by lasley** [comment]
I am able to replicate this issue on Runbot, although the check that doesn't seem to stick is the "Include the production potential based on" one - the "Exclude incoming goods" seems fine. The following traceback is in the run log:

```
2017-08-12 12:15:25,794 167 ERROR openerp_test odoo.http: Exception during JSON request handling.
Traceback (most recent call last):
File "/.repo_requirements/odoo/odoo/http.py", line 640, in _handle_exception
return super(JsonRequest, self)._handle_exception(exception)
File "/.repo_requirements/odoo/odoo/http.py", line 677, in dispatch
result = self._call_function(**self.params)
File "/.repo_requirements/odoo/odoo/http.py", line 333, in _call_function
return checked_call(self.db, *args, **kwargs)
File "/.repo_requirements/odoo/odoo/service/model.py", line 101, in wrapper
return f(dbname, *args, **kwargs)
File "/.repo_requirements/odoo/odoo/http.py", line 326, in checked_call
result = self.endpoint(*a, **kw)
File "/.repo_requirements/odoo/odoo/http.py", line 935, in __call__
return self.method(*args, **kw)
File "/.repo_requirements/odoo/odoo/http.py", line 506, in response_wrap
response = f(*args, **kw)
File "/home/odoo/odoo-10.0/addons/web/controllers/main.py", line 885, in call_kw
return self._call_kw(model, method, args, kwargs)
File "/home/odoo/odoo-10.0/addons/web/controllers/main.py", line 877, in _call_kw
return call_kw(request.env[model], method, args, kwargs)
File "/.repo_requirements/odoo/odoo/api.py", line 681, in call_kw
return call_kw_multi(method, model, args, kwargs)
File "/.repo_requirements/odoo/odoo/api.py", line 672, in call_kw_multi
result = method(recs, *args, **kwargs)
File "/.repo_requirements/odoo/odoo/models.py", line 5506, in onchange
newval = record[name]
File "/.repo_requirements/odoo/odoo/models.py", line 5214, in __getitem__
return self._fields[key].__get__(self, type(self))
File "/.repo_requirements/odoo/odoo/fields.py", line 872, in __get__
self.determine_draft_value(record)
File "/.repo_requirements/odoo/odoo/fields.py", line 992, in determine_draft_value
self._compute_value(record)
File "/.repo_requirements/odoo/odoo/fields.py", line 927, in _compute_value
getattr(records, self.compute)()
File "/home/odoo/build/OCA/stock-logistics-warehouse/stock_available/models/product_template.py", line 70, in _compute_potential_qty
tmpl.potential_qty = res[tmpl.id]['potential_qty']
KeyError: 'potential_qty'
```

**Comment by ofrmax** [comment]
Hi on my side both checkboxes not working

**Comment by ofrmax** [comment]
Any news on this problem ? and how we can solve it ?

**Comment by ofrmax** [comment]
HI Guys, I would really like to use this addon and I cannot really help on the coding side :( Anyone know how to solve this issue ?

**Comment by rousseldenis** [comment]
@lasley The KeyError error should be fixed in another PR : https://github.com/OCA/stock-logistics-warehouse/pull/305/ (even if it is not the PR subject).

@ofrmax The screen you are redirected is the next action returned by the configuration workflow. In my case (and I think it's yours), it wants to install a module which is not present on your system (me : stock_availlable_mrp), so Odoo redirects you to the app page.

https://github.com/odoo/odoo/blob/ce9bbf6c10962e64138c04ca34b7485084e4cdaa/odoo/addons/base/res/res_config.py#L562

**Comment by LoisRForgeFlow** [comment]
Fixed at #305
